### ISSUE TITLE
[A11y] Color contrast issues on package details page, links on the selected version

### ISSUE THREAD

**Issue by joelverhagen** [NuGet/NuGetGallery#8091]
Issue: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (color-contrast - https://dequeuniversity.com/rules/axe/3.5/color-contrast?application=msftAI)

Target application: `NuGet Gallery | Newtonsoft.Json 12.0.3 - https://www.nuget.org/packages/Newtonsoft.Json/`

Element paths: `a[title="\31 2\.0\.3"]`

Snippet: `<a href="/packages/Newtonsoft.Json/12.0.3" title="12.0.3"> 12.0.3 </a>`

Element path: `.bg-info > td:nth-child(4) > a`

Snippet: `<a href="/packages/Newtonsoft.Json/12.0.3/Manage">Listed</a>`

![image](https://user-images.githubusercontent.com/94054/86853070-af663980-c06a-11ea-9f94-4ab49b43c88a.png)

How to fix: 
Fix any of the following:
 Element has insufficient color contrast of 3.77 (foreground color: #337ab7, background color: #d9edf7, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1

Environment: Chrome version 83.0.4103.116

**Comment by ryuyu** [comment]
It looks like changing the active row color for links to either not be a link (black) or to use the hovered value (#23527c) is sufficient contrast.
